src/index.html

```diff
@@ -10,6 +10,7 @@
   <body>
     <app-root></app-root>
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-98841695-4"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
```

Comment:
I think we can use the node package directly in package.json: https://www.npmjs.com/package/@google-cloud/storage

Reply:
I might be doing something wrong I just gave that a go and it reminded me of why I switched of the just using `api.js`. When I add `@google-cloud/storage`, it gives me a bunch of compile errors as there are missing dependencies. Here's a sample of the output. 
```
Module not found: Error: Can't resolve 'tls' in '/ng-app/node_modules/https-proxy-agent/dist'
resolve 'tls' in '/ng-app/node_modules/https-proxy-agent/dist'
  Parsed request is a module
  using description file: /ng-app/node_modules/https-proxy-agent/package.json (relative path: ./dist)
....
Module not found: Error: Can't resolve 'tls' in '/ng-app/node_modules/tunnel-agent'
resolve 'tls' in '/ng-app/node_modules/tunnel-agent'
...
Module not found: Error: Can't resolve 'zlib' in '/ng-app/node_modules/@google-cloud/storage/build/src'
resolve 'zlib' in '/ng-app/node_modules/@google-cloud/storage/build/src'
...
ERROR in ./node_modules/teeny-request/build/src/index.js
Module not found: Error: Can't resolve 'stream' in '/ng-app/node_modules/teeny-request/build/src'
resolve 'stream' in '/ng-app/node_modules/teeny-request/build/src'
  Parsed request is a module
```

The way I used felt old school but it didn't have the weight of adding all of those extra dependencies. If that isn't a concern I can try to work through those errors and get it going :)

Reply:
Hm, that's strange. This worked for me:
```
> npm add @google-cloud/storage
> npm install
> npm run
```

Reply:
hmm in that case I will blame my env. I'll give it a go with the commands you have there. Thanks for testing on your box :)

Reply:
Gave it a go this morning and it started off positively on a different machine but then I ended up getting the same errors. Pushed the latest code, maybe you'll be able to catch something I missed.

```
ERROR in ./node_modules/tunnel-agent/index.js
Module not found: Error: Can't resolve 'tls' in '/Users/work/Documents/release-notes/node_modules/tunnel-agent'

ERROR in ./node_modules/@google-cloud/storage/build/src/file.js
Module not found: Error: Can't resolve 'zlib' in '/Users/work/Documents/release-notes/node_modules/@google-cloud/storage/build/src'

ERROR in ./node_modules/request/request.js
Module not found: Error: Can't resolve 'zlib' in '/Users/work/Documents/release-notes/node_modules/request'
```

Reply:
After debugging this a bit it seems like @google-cloud/storage package is not meant to be used client side (ref: https://github.com/googleapis/google-api-nodejs-client/issues/2211#issuecomment-545137289).

Instead of using this package, we could directly fetch the `release-notes-index.json` using its url https://gcsweb.k8s.io/gcs/kubernetes-release/release/release-notes-index.json (thanks @saschagrunert for the link). And then using the index we can build the `assets` array.

The only blocker here seems that the above URL doesn't support CORS which is required to make AJAX calls from [relnotes.k8s.io](https://relnotes.k8s.io)